google-account-plugin/src/com/google/cloud/tools/intellij/login/ui/GoogleLoginUsersPanel.java

```diff
@@ -186,11 +187,9 @@ public void mouseDragged(MouseEvent event) {
     boolean noUsersAvailable =
         (listModel.getSize() == 1) && (listModel.get(0) instanceof NoUsersListItem);
     JButton addAccountButton = new JButton(noUsersAvailable ? SIGN_IN : ADD_ACCOUNT);
-    addAccountButton.addActionListener(new ActionListener() {
-      @Override
-      public void actionPerformed(ActionEvent event) {
-        Services.getLoginService().logIn();
-      }
+    addAccountButton.addActionListener(event -> {
+      SwingUtilities.getWindowAncestor(this).dispose();
```

Comment:
1. Open IDEA.
2. Click the profile image button to open the Google Account popup.
3. Click the "Sign In" button or the "Add Account" button, whichever is appropriate for your account state.
4. Observe the Google Account popup remains visible on top of the opened browser login window. See the picture below.

![screen shot 2017-05-17 at 12 07 44 pm](https://cloud.githubusercontent.com/assets/14352762/26171375/9b04af2a-3af9-11e7-8824-d3ec68a2e33f.png)